config/menu.ts

```diff
@@ -48,6 +48,13 @@ export default function Menu(conference: Conference, dates: Dates): { Top: MenuI
     })
   }
 
+  if (dates.IsInProgress) {
```

Comment:
Per comment in teams, could be useful if this was available before-hand?

Reply:
I'll change the check to make this available in the week leading up to the big day instead.